packages/cfpb-atomic-component/src/mixins/EventObserver.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * EventObserver
+ * @class
+ *
+ * @classdesc Used for creating an object
+ *   that can be used to dispatch and listen to custom events.
+ * @returns {Object} An EventObserver instance.
+ */
+function EventObserver() {
```

Comment:
I think maybe we didn't because of lack of IE11 support, though babel should take care of it. This might be better in its own PR once this has been released and the version in cf.gov is replaced https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/js/modules/util/EventObserver.js

Reply:
Hmmm yeah and updating just a single file to a class isn't super pretty. Maybe a future PR can update them all.